mappings/net/minecraft/potion/Potion.mapping

```diff
@@ -1,7 +1,7 @@
 CLASS none/age net/minecraft/potion/Potion
 	FIELD a REGISTRY Lnone/cs;
-	FIELD b id Lnone/kp;
-	FIELD c maxRawId I
+	FIELD b WATER_TEX Lnone/kp;
```

Comment:
I remember seeing a lot of texture locations defined as `BG_TEX` and similar, as such I thought that was the way to do it. I'd say the type already shows that it's a location.

Let's vote on it? :+1: for `WATER_TEX_LOC` suffix, :-1: for `WATER_TEX`?